packages/pwa-kit-build/src/configs/webpack/config.js

```diff
@@ -277,12 +277,18 @@ const renderer =
                     new CopyPlugin({
                         patterns: [
                             {from: 'app/static/', to: 'static/'},
+                        ]
+                    }),
+
+                    new CopyPlugin({
+                        patterns: [
                             {
                                 from: 'config/',
                                 to: 'config/',
                                 globOptions: {
                                     ignore: ['**/local.*']
-                                }
+                                },
+                                noErrorOnMissing: true,
```

Comment:
We've said that these configs should be optional and fall back to `package.json`. Without this change, `webpack` failed where `cosmiconfig` didn't.